node/bindnode/example_test.go

```diff
@@ -80,3 +82,104 @@ func ExamplePrototype_onlySchema() {
 	// Output:
 	// {"Friends":["Sarah","Alex"],"Name":"Michael"}
 }
+
+func ExampleWrap_withSchemaUsingStringjoinStruct() {
+	ts := schema.TypeSystem{}
+	ts.Init()
+	ts.Accumulate(schema.SpawnString("String"))
+	ts.Accumulate(schema.SpawnStruct("FooBarBaz",
+		[]schema.StructField{
+			schema.SpawnStructField("foo", "String", false, false),
+			schema.SpawnStructField("bar", "String", false, false),
+			schema.SpawnStructField("baz", "String", false, false),
+		},
+		schema.SpawnStructRepresentationStringjoin(":"),
+	))
+	schemaType := ts.TypeByName("FooBarBaz")
+
+	type FooBarBaz struct {
+		Foo string
+		Bar string
+		Baz string
+	}
+	fbb := &FooBarBaz{
+		Foo: "x",
+		Bar: "y",
+		Baz: "z",
+	}
+	node := bindnode.Wrap(fbb, schemaType)
+
+	// Take the representation of the node, and serialize it.
+	nodeRepr := node.Representation()
+	var buf bytes.Buffer
+	dagjson.Encode(nodeRepr, &buf)
+
+	// Output how this was serialized, for the example.
+	fmt.Fprintf(os.Stdout, "json: %s\n", buf.Bytes())
```

Comment:
nit: for the sake of keeping the example simpler, you could just dagjson.Encode straight to os.Stdout, printing `json:` in the line above or in the same line via Printf